src/View/Helper/FormHelper.php

```diff
@@ -285,20 +285,10 @@ public function control($fieldName, array $options = [])
         }
 
         if ($options['help']) {
-            if (is_string($options['help'])) {
-                $options['help'] = $this->templater()->format(
-                    'help',
-                    ['content' => $options['help']]
-                );
-            } elseif (is_array($options['help'])) {
```

Comment:
Why did you remove the ability to set 'help' key to an array? This feature was recently added on master as someone wanted to add extra attributes to the tag used for help message.